src/content/docs/en/guides/deploy.mdx

```diff
@@ -41,9 +43,23 @@ Some hosts will have their own command line interface (CLI) you can install glob
 
 1. Install your host's CLI globally, for example:
 
-    ```bash
-    npm install --global netlify-cli
-    ```
+<PackageManagerTabs>
+    <Fragment slot="npm">
+      ```shell
+      npm install --global netlify-cli
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm add --global netlify-cli
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn global add netlify-cli
+      ```
+    </Fragment>
+  </PackageManagerTabs>
```

Comment:
Changes involving ordered list currently are not looking right.

As an example: https://deploy-preview-5305--astro-docs-2.netlify.app/en/guides/deploy/
![image](https://github.com/withastro/docs/assets/11240579/a8a1f591-6ab6-4243-92c2-b63071aaaf0c)

The CLI command area should be a child of the ordered list, currently it's being rendered as a sibling element between two ordered lists.
You can add indentation to make it render as a child:

```markdown
    <PackageManagerTabs>
        <Fragment slot="npm">
          ```shell
          npm install --global netlify-cli
          ```
          </Fragment>
          <Fragment slot="pnpm">
          ```shell
          pnpm add --global netlify-cli
          ```
          </Fragment>
          <Fragment slot="yarn">
          ```shell
          yarn global add netlify-cli
          ```
        </Fragment>
    </PackageManagerTabs>
```